tests/test_lldpSyncDaemon.py

```diff
@@ -54,15 +65,15 @@ def setUp(self):
 
     def test_parse_json(self):
         jo = self.daemon.parse_update(self._json)
-        print(json.dumps(jo, indent=3))
+        print((json.dumps(jo, indent=3)))
```

Comment:
Unnecssary extra set of parentheses. Another overly-cautious artifact of `2to3`.